auto_update_pricelist/models/partner.py

```diff
@@ -33,9 +33,11 @@ class ResPartner(models.Model):
     )
 
     @api.model
-    def _get_year_dates(self):
+    def _get_year_dates(self, year):
         start_date = datetime.now().date().replace(month=1, day=1)
```

Comment:
Can we use fields.Datetime.now instead of using datetime.now() ?

I think it was by our code but you can correct it.